### PR TITLE
Add an example code for Android

### PR DESCRIPTION
**Pull request by hnvn**
This example contains an implementation of CvCameraPreview that fetches image data from physical camera, convert data to Mat, expose an interface for OpenCV processing, then render processed image data in a SurfaceView.

### CODE REVIEW

**Comment by saudet** [comment]
Looks good! Just a couple of things:
- Remove the `.idea` folder and/or add it to `.gitignore`
- Rename the `com.javacv` package to something sensible like `org.bytedeco.javacv`
- Since we already have a `JavaCV-android-example` project, could you either:
  - Update that project with your files
  - Use another name for this project like `javacv-android-display` or something
- Fix typos (mainly `exmaple` -> `example`)

Thanks!

**Comment by saudet** [comment]
Thanks for your changes, let me know when you are done! Thanks

**Comment by hnvn** [comment]
I already finished my work for example

**Comment by saudet** [comment]
Ok, thank you, but could you also change the things I mention in my comment above?

**Comment by hnvn** [comment]
I changed something according to your mentions. You can review and tell me any still existing problems.

**Comment by saudet** [comment]
Great! But yes, please fix the following:
1. Rename `JavaCv-android-preview-exmaple` to `javacv-android-preview-example` (although the other Android example also does preview, so we might want to have a better name...)
2. Remove all the `.gitignore` files that are not in that directory
3. Update the `com.javacv` reference to `org.bytedeco.javacv`

The rest looks good so, once this is done, I'll merge this! Thanks

File: JavaCv-android-preview-exmaple/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<resources>
+    <string name="app_name">JavaCvAndroidExmaple</string>
```
Comment (saudet) [inline JavaCv-android-preview-exmaple/app/src/main/res/values/strings.xml]: Also update the `app_name` to the same value as the directory name. Thanks!

**Comment by hnvn** [comment]
Done! Everything seems Ok. Please check again and tell me anything wrong.

**Comment by saudet** [comment]
Ok, thanks! Looks OK, just one last thing: Please remove the .gitignore file in the top directory. Feel free to use one in your project directory though.

**Comment by hnvn** [comment]
Done

**Comment by saudet** [comment]
And the directory name is a bit long. Let's settle for "javacv-android-camera-preview", would that be OK?

**Comment by hnvn** [comment]
It's ok now

**Comment by saudet** [comment]
Great! Awesome. Thank you very much

**Pull request merged**